packages/@o3r/localization/schematics/localization-base/index.ts

```diff
@@ -463,11 +469,6 @@ export function updateLocalization(options: { projectName?: string | null | unde
   ]);
 }
 
-/**
- *
- * @param options
- * @param options.projectName
- */
 function updateI18nFn(options: { projectName?: string | undefined }): Rule {
```

Comment:
why did you remove the jsdoc ?

Reply:
It's a private function so jsdoc is not mandatory and this doc was not adding any values